content/docs/dvclive/how-it-works.md

```diff
@@ -68,23 +70,67 @@ model.pt
 model.pt.dvc
 ```
 
-## Track the results
+## Tracking with Git+DVC
 
 DVCLive expects each run to be tracked by Git, so it will save each run to the
-same path and overwrite the results each time. If you
-[run experiments](/doc/user-guide/experiment-management/running-experiments)
-using DVC <abbr>pipelines</abbr>, DVC will track them for you so you don't have
-to Git commit each run. <abbr>DVC experiments</abbr> are Git commits that DVC
-can find but that don't clutter your Git history or create extra branches in
-your repo.
+same path and overwrite the results each time. Include
+[`save_dvc_exp=True`](/doc/dvclive/api-reference/live#parameters) to auto-track
+as a <abbr>DVC experiment</abbr>. DVC experiments are Git commits that DVC can
+find but that don't clutter your Git history or create extra branches.
 
-If you don't have a DVC pipeline, you can include
-[`save_dvc_exp=True`](/doc/dvclive/api-reference/live#parameters) to save the
-results as a DVC experiment.
+### Track large artifacts with DVC
 
-When using `Live.log_artifact("model.pt")`, DVCLive will
+Models and data are often large and aren't easily tracked in Git.
+`Live.log_artifact("model.pt")` will
 [cache](/doc/start/data-management/data-versioning) the `model.pt` file with DVC
-to avoid tracking large artifacts in Git. It will generate a `model.pt.dvc`
-metadata file, which you should track in Git. You can
-[retrieve](/doc/start/data-management/data-versioning#retrieving) the artifact
-from the Git commit.
+and make Git ignore it. It will generate a `model.pt.dvc` metadata file, which
+can be tracked in Git and becomes part of the experiment. With this metadata
+file, you can [retrieve](/doc/start/data-management/data-versioning#retrieving)
+the versioned artifact from the Git commit.
+
+### Run with DVC
+
+Experimenting in Python interactively (like in notebooks) is great for
+exploration, but eventually you may need a more structured way to run
+reproducible experiments (for example, running a parallelized hyperparameter
+search). By configuring DVC <abbr>pipelines</abbr>, you can
+[run experiments](/doc/user-guide/experiment-management/running-experiments)
+with `dvc exp run`.
+
+DVCLive prints instructions for how to start:
+
+```
+To run with DVC, add this to `/Users/sarah/myproject/dvc.yaml`:
+stages:
+  dvclive:
+    cmd: <python my_code_file.py my_args>
+    deps:
+    - <my_code_file.py>
+    outs:
+    - dvclive/params.yaml:
```

Comment:
@sroy3 could you please remind what code we generate now? It would be great to consolidate this a bit ...

Reply:
This is what we generate from the stage name, command and script given by the user.

```
# Read about DVC pipeline configuration (https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#stages)
# to customize your stages even more
stages:
  ${stageName}:
    cmd: ${command} ${relativeScript}
    deps:
      - ${relativeScript}```

Reply:
thanks @sroy3 !

Reply:
@dberenbaum should we align this (e.g. add `outs` - what do you think?)

Reply:
🤔 I don't think we need to block or align too much yet but it's worth continuing to discuss as we get feedback. In DVCLive, it's hard to figure out the deps/command (although we might be able to do more to fill these in sometimes). In VS Code, it's cumbersome to ask for outs since it would be as easy to write it directly in dvc.yaml.

Reply:
Yep, I mean specifically in case of DVCLive, when we know (can we?) that's it is DVCLive generated directory.

Reply:
> when we know (can we?)

I can't think of how 😅 but maybe I'm missing something here